vllm/core/block/prefix_caching_block.py

```diff
@@ -351,6 +376,9 @@ def __init__(
         self._prev_block = prev_block
         self._cached_content_hash: Optional[int] = None
         self._prefix_caching_allocator = prefix_caching_allocator
+        self.computed = False
+        self.num_hashed_tokens = 0
```

Comment:
Nit: Instead of manually setting this when marking blocks as computed, you should be able to do something like:
```
    @property
    def num_tokens_total(self) -> int:
        return self._prev_block.num_tokens_total + len(self.token_ids)
```
You'll have to account for the _prev_block == None case.

Reply:
I didn't get the num_tokens_total usage here...